ansible/tasks/oauth2-proxy.yml

```diff
@@ -46,16 +46,17 @@
 
 # Same thing but now for admin only oauth access
 
-- name: "copy allowed users file"
+- name: "copy authorized users file"
   template:
-    src: "templates/etc/oauth2_proxy/allowed-users"
-    dest: "/etc/oauth2_proxy/allowed-users"
+    src: "templates/etc/oauth2_proxy/authorized-users"
+    dest: "/etc/oauth2_proxy/authorized-users"
```

Comment:
If we're making the entire contents of the file a variable we can remove the template and directly place the contents of the file with the [`copy` module] like this:

```yaml
- name: ""
  copy:
    dest: "/etc/oauth2_proxy/authorized-users"
    contents: "{{ secret_oauth2_proxy.authorized_users }}"
```

[`copy` module]: https://docs.ansible.com/ansible/latest/modules/copy_module.html#copy-module